ReactAndroid/build.gradle

```diff
@@ -391,6 +393,7 @@ android {
         targetSdkVersion(28)
         versionCode(1)
         versionName("1.0")
+        ndkVersion "20.1.5948944"
```

Comment:
Are you sure this goes here (inside of `defaultConfig`)? Same question for the other file modified.

Reply:
Yes, the reason I put it in the defaultConfig is yesterday I was playing around Gradle and trying to answer Kevin suggestion on whether or not we can `put all these "blessed versions" in one place so that we minimize repetition` but I can't seem to find a way because ReactAndroid, Build Gradle is a different entity. Do you have any suggestions? I will put it directly inside `android` for now. 

Thanks for the quick review btw @friederbluemle. Much appreciated